integration/feature/init/src/MillInitSbtTests.scala

```diff
@@ -0,0 +1,332 @@
+package mill.integration
+
+import mill.constants.Util
+import mill.integration.testMillInit
+import utest.*
+
+private def bumpSbtTo1107(workspacePath: os.Path) =
+  // bump sbt version to resolve compatibility issues with lower sbt versions and higher JDK versions
+  os.write.over(workspacePath / "project" / "build.properties", "sbt.version = 1.10.7")
+
+// relatively small libraries
+
+private val scalaPlatforms = Seq("js", "jvm", "native")
+
+object MillInitSbtLibraryExampleTests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 21 KB
+    - sbt 1.5.2
+     */
+    val url = "https://github.com/scalacenter/library-example/archive/refs/tags/v1.0.1.zip"
+
+    test - integrationTest(url)(
+      _.testMillInit(
+        expectedCompileTasks = Some(SplitResolvedTasks(Seq("compile"), Seq())),
+        expectedTestTasks = None // scalaprops not supported in `TestModule`
```

Comment:
Maybe we can pick another example repo that has a supported test framework? e.g. https://github.com/scala/scala3-example-project seems to use munit which is supported by `TestModule.Munit`